sources/pmwebapi.go

```diff
@@ -110,8 +110,27 @@ func typeLabel(units string, pcpType string, name string) string {
 	} else {
 		name += unit
 	}
+	return name
+}
+
+func fixNaming(name string) string {
 	name = strings.Replace(name, "count / sec", "count_per_second", -1)
-	return strings.Replace(name, ".", "_", -1)
+	name = strings.Replace(name, "Kbyte", "kilobyte", -1)
```

Comment:
Note that when you convert these to switchcases, you will no longer require every single one of these items. Some of the later ones are a result of these earlier actions. I will let you figure out which ones you need to keep.